nix/default.nix

```diff
@@ -63,6 +63,7 @@ pkgs.haskell-nix.cabalProject {
       config.treefmt.build.wrapper
       pkgs.torch
       pkgs.torch.dev
+      pkgs.ghcid
```

Comment:
Does not add it to shell tools because https://github.com/haskell-fswatch/hinotify/pull/39